src/components/GalleryFilter.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react'
+
+export default class GalleryFilter extends Component {
+    render() {
+        const handleFunction = this.props.handleFunction
+        const title = this.props.title
+        const filteredData = this.props.filteredData
```

Comment:
a neat way to DRY this out is by destructuring this.props like so:
```js
{ handleFunction,
  title,
  filteredData } = this.props;
```

Reply:
Whoa! Thanks for the recommendation, I forgot how nice destructuring can be.